src/Microsoft.DotNet.Darc/src/DarcLib/Models/Darc/DownloadedAsset.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Maestro.Client.Models;
+
+namespace Microsoft.DotNet.DarcLib.Models.Darc
+{
+    public class DownloadedAsset
+    {
+        /// <summary>
+        /// Asset that was downloaded.
+        /// </summary>
+        public Asset Asset { get; set; }
+        /// <summary>
+        /// Source location (uri) where the asset came from.
+        /// </summary>
+        public string SourceLocation { get; set; }
+        /// <summary>
+        /// Target location where the asset was downloaded to for the release style layout
+        /// </summary>
+        public string ReleaseLayoutTargetLocation { get; set; }
```

Comment:
Is this added to the manifest anywhere? That would be a useful bit of information to keep in the manifest, especially when we use relative paths

Reply:
it's in there but without a name;  this is just a type I moved to DarcLib, not new.  It's a full path to a single asset though, not the base value of such.

Reply:
(one of the two properties that generate these:)

```
          "targets": [
            "arcade\\20201009.8\\nonshipping\\assets\\symbols\\Microsoft.DotNet.Build.Tasks.Feed.6.0.0-beta.20509.8.symbols.nupkg",
            "nonshipping\\assets\\symbols\\Microsoft.DotNet.Build.Tasks.Feed.6.0.0-beta.20509.8.symbols.nupkg"
          ],
```

Reply:
chatted on Teams, I pushed  change that puts the output folder into the manifest in all cases, looks like this:

```
{
  "outputPath": "f:\\some\\path",
  "builds": [
    {
      "repo": "https://github.com/dotnet/somerepo",
      "commit": "ba32978d76r590e1a5cde6bf54eb4e9b8d212c73",
      "branch": "refs/heads/master",
      "produced": "2020-10-09T19:35:12+00:00",
      "buildNumber": "20201009.2",
      "barBuildId": 12345,
      "channels": [
        {
          "id": 9,
          "name": ".NET Eng - Validation"
        }
      ],
...
```